packages/video_player_avplayer/tizen/src/media_player.cc

```diff
@@ -0,0 +1,621 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "media_player.h"
+
+#include <dlfcn.h>
+
+#include "log.h"
+
+static std::string RotationToString(player_display_rotation_e rotation) {
+  switch (rotation) {
+    case PLAYER_DISPLAY_ROTATION_NONE:
+      return "PLAYER_DISPLAY_ROTATION_NONE";
+    case PLAYER_DISPLAY_ROTATION_90:
+      return "PLAYER_DISPLAY_ROTATION_90";
+    case PLAYER_DISPLAY_ROTATION_180:
+      return "PLAYER_DISPLAY_ROTATION_180";
+    case PLAYER_DISPLAY_ROTATION_270:
+      return "PLAYER_DISPLAY_ROTATION_270";
+  }
+  return std::string();
+}
+
+MediaPlayer::MediaPlayer(flutter::BinaryMessenger *messenger,
+                         void *native_window)
+    : VideoPlayer(messenger), native_window_(native_window) {
+  media_player_proxy_ = std::make_unique<MediaPlayerProxy>();
+}
+
+MediaPlayer::~MediaPlayer() { Dispose(); }
+
+int64_t MediaPlayer::Create(const std::string &uri, int drm_type,
+                            const std::string &license_server_url,
+                            bool is_prebuffer_mode) {
+  LOG_INFO("[MediaPlayer] uri: %s, drm_type: %d.", uri.c_str(), drm_type);
+
+  if (uri.empty()) {
+    LOG_ERROR("[MediaPlayer] The uri must not be empty.");
+    return -1;
```

Comment:
'Create' will return player id, and '-1' may be considered as player id.

Reply:
Fixed.